### ISSUE TITLE
Shut down reliability judgements

### ISSUE THREAD

**Issue by cormacanderson** [lingdb/CoBL#229]
The reliability judgement system is in danger of recreating issues we have being trying to eradicate. For this reason, we have to try to shut it down. This is a multi-step process, beginning with the abolition of reliability judgements under `lexical citation`. Upon completion of https://github.com/lingdb/CoBL/issues/228, we will shut down reliability judgements also for `cognate citation`. 

While we are now focusing on the `Jena200` meaning list and the `Current` meaning list, I would like to make sure we do not lose data outside of these lists. The first step below means that all lexemes which have been excluded in previous rounds of data entry are marked `Not Swh`. The second will allow us to target the meanings in the `all` list which may have been reassigned as Swadesh in the current round of data entry (there won't be many).
- [x] For all lexemes marked `lex excl`, mark them `Not Swh`
- [x] Compute statistics for + lex. ecl. - Not swh for the `Current` language list and print below

The next step ensures that entries reassigned as Swadesh in the `Current` language list during this round of data entry are guaranteed to go through to the export:
- [x] Set the reliability judgements for all `lexical citation` as "high"

The next steps should shut down reliability judgements for `lexeme citation`
- [x] Allow `lexical citation` to be entered without a reliability judgement
- [x] Hide reliability judgements for `lexical citation`

Further steps depend on the elimination of loanwords https://github.com/lingdb/CoBL/issues/228:
- [x] Set all reliability judgements for all `cognate citation` as "high"
- [x] Allow `cognate citation` to be entered without a reliability judgement
- [x] Hide reliability judgements for `cognate citation`

The last step will modify the export code to remove mention of reliability judgements:
- [x] Exclude lexemes only on the basis of `Not Swh` (and `Loanword Event`)

**Comment by runjak** [comment]
This sounds like a good roadmap. Unfortunately I'm not entirely clear which parts I can/should tackle right away.
My current understanding is that the tasks I can tackle immediately are all that are the ones depending on #228. Is that correct?
I would aim to deploy each step sequentially in that case.

**Comment by cormacanderson** [comment]
The steps should be sequential and the first five can be done, as far as I assume,  before #228 is completed.

**Comment by runjak** [comment]
I've written code to mark Lexemes as `not_swadesh_term` where `is_excluded_lexeme` holds which will soon be deployed on the server. I've made a list of lexemes that will be marked in case we discover that we need it: 
[changedLexemes.txt](https://github.com/lingdb/CoBL/files/375650/changedLexemes.txt)

**Comment by runjak** [comment]
I've seen that there is indeed a checkbox in this issue that asks to compute the Lexemes that satisfy `is_excluded_lexeme` but are not marked as `not_swadesh_term`.
While they will be marked soon automagically it makes sense to compute a list of them that is more cleaned up and recent than the one in my last comment:
[wantedLexemes.txt](https://github.com/lingdb/CoBL/files/375709/wantedLexemes.txt)

**Comment by runjak** [comment]
I've deployed an update to the server taking care of these items:
* For all lexemes marked `lex excl`, mark them `Not Swh`
* Compute statistics for + lex. ecl. - Not swh for the `Current` language list and print below
* Set the reliability judgements for all `lexical citation` as "high"
* Allow `lexical citation` to be entered without a reliability judgement
* Hide reliability judgements for `lexical citation`

As far as I understand I can already work on:
* Exclude lexemes only on the basis of `Not Swh` (and `Loanword Event`)

For the other two I'll wait on #228 as specified.

**Comment by PaulHeggarty** [comment]
I’m downrating the urgency to level 1, because the remaining check box points within this issue cannot be solved until #228 is fixed, which won’t be until sometime in the first week of August, at the earliest.

**Comment by PaulHeggarty** [comment]
It is now some time in the first week of August, so I'm uprating the urgency again.

**Comment by cormacanderson** [comment]
I have modified the list of tasks. However, please do not proceed with this until https://github.com/lingdb/CoBL/issues/228 has been closed.

**Comment by cormacanderson** [comment]
Also https://github.com/lingdb/CoBL/issues/262 should be dealt with before this is closed

**Comment by runjak** [comment]
I've got a question about the last item in this issue:
> Exclude lexemes only on the basis of Not Swh (and Loanword Event)

If I'm not mistaken the `Loanword Event` is a property of a cognate set that a lexeme belongs to whereas `Not Swh` is a property of a Lexeme.

At the moment our export code [exludes](https://github.com/lingdb/CoBL/blob/c6109d0dc8d63f752300c7823ae6bfb61b7c41e2/ielex/lexicon/views.py#L370-L374) all lexemes that are marked as `not_swadesh_term`.

Do I understand correctly that we want an option to exclude all lexemes that belong to a cognate class that is marked as loan event?

**Comment by cormacanderson** [comment]
Exactly. It used to be that records with "Excluded" and "Loanword" reliability judgements could be and usually were (in theory if not always in practice ;) ) excluded from the export. We need to recreate this functionality. There might be further options that we want to include within the class of loanwords, but for now, that is enough.

**Comment by runjak** [comment]
I've implemented the exclusion of lexemes based on their cognate class and added a new checkbox to the nexus export form for this:
![screenshot_2016-08-11_15-10-32](https://cloud.githubusercontent.com/assets/5417642/17589567/c9fe840e-5fd5-11e6-942f-47b42dfef5d7.png)

The time an export takes is still a problem that I'd like to solve, but not part of this issue.
This issue will be closed with the next deployment.

**Comment by PaulHeggarty** [comment]
Sorry, but there is something wrong big-style in the instruction to Jakob here.  Now that cognate sets are specified as of a loanword event type, and the lexemes are **not** included any more in the same cognate set as the source, then we absolutely **do** want to include loanword-event cognate sets as standard.  
This new checkbox is nonetheless useful, because it allows us to do exports with an even more radical approach to loanwords, but it needs to be renamed and explained very differently, and set to not exclude loanword event cognate sets by default.  
If we want to reproduce in effect the old IELex system, then we need on the contrary a button with an option to merge loanword cognate sets into their source cognate sets provided they are in the same meaning, but I don’t see any utility for that for now, so it's not to be done yet!

**Comment by cormacanderson** [comment]
I think there might be some misunderstanding here. I saw it more that we want the ability to not consider loanwords (here loanword cognate sets), as there was in the old system. I would have thought that for any export it is first necessary to choose what and what not to exclude, so wasn't really thinking in terms of defaults. I'm not sure if the ability to merge loanword cognate sets into their source cognate sets actually reproduces the old system, because that is not, as I understand it, how the old system worked. In the old system, loanword were marked as 0 for a given cognate set, not treated as simple members of it (i.e. marked 1) which is what the effect of merging would have.  
We might want to consider giving the option to exclude sets on the basis of the other boolean fields on the `1 Mng, All Cog Sets` form, i.e. "Ideophonic", "Pll. Derivation", "Dubious Set", and "Parallel Loan".

**Comment by runjak** [comment]
OK - this looks like it would be helpful to discuss this in person together.

I'm not clear what to do next with this issue:
* I think making adjustments to the nexus export would warrant a separate issue
* And I'm not clear what task I should infer from the last two comments.

Therefore I would suggest closing this issue and opening a new one to modify the exports once we know what we want the export to do.

**Comment by runjak** [comment]
Closing this as nothing further happened in the last 11 days.
